VimeoNetworking/Sources/Models/VIMQuantityQuota.m

```diff
@@ -39,8 +39,8 @@ @implementation VIMQuantityQuota
 
 - (void)didFinishMapping
 {
-    self.canUploadHd = [self.hd boolValue];
-    self.canUploadSd = [self.sd boolValue];
+    self.canUploadHd = [self.hd respondsToSelector:@selector(boolValue)] ? [self.hd boolValue] : NO;
+    self.canUploadSd = [self.sd respondsToSelector:@selector(boolValue)] ? [self.sd boolValue] : NO;
```

Comment:
we don't know what the exact cause is yet, but the API was essentially sending back empty objects as the value instead of bool, so the object mapper converted those objects into NSDictionary objects which don't respond to boolValue.  The closest we've been to seeing this in house was a week or two ago when one of the VMs was broken and sending out bad data for all requests.